commands/changesets.md

```
@@ -1,667 +1,156 @@
+   * One to two sentences max.
+   * Lead with the user-visible outcome (what was added/changed/fixed).
+   * Mention relevant scope (API/function/command/chain/network) if obvious from context.
+   * Avoid commit jargon; no issue numbers unless essential.
+   * Do not restate diff stats.
+   * Keep it tense-consistent and crisp.
 
-# Check for staged changes
-git diff --cached --name-only
+4. **Output format (strict):**
+   * Output **only** the chainset content:
+     ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)